src/main/scala/firrtl/passes/ExpandWhens.scala

```diff
@@ -98,90 +89,74 @@ object ExpandWhens extends Pass {
       // defaults ideally would be immutable.Map but conversion from mutable.LinkedHashMap to mutable.Map is VERY slow
       def expandWhens(
           netlist: LinkedHashMap[WrappedExpression, Expression],
-          defaults: Seq[collection.mutable.Map[WrappedExpression, Expression]],
-          p: Expression)
-          (s: Statement): Statement = {
-        s match {
-          case w: DefWire =>
-            getFemaleRefs(w.name, w.tpe, BIGENDER) foreach (ref => netlist(ref) = WVoid())
-            w
-          case r: DefRegister =>
-            getFemaleRefs(r.name, r.tpe, BIGENDER) foreach (ref => netlist(ref) = ref)
-            r
-          case c: Connect =>
-            netlist(c.loc) = c.expr
-            EmptyStmt
-          case c: IsInvalid =>
-            netlist(c.expr) = WInvalid()
-            EmptyStmt
-          case s: Conditionally =>
-            val memos = ArrayBuffer[Statement]()
+          defaults: Seq[mutable.Map[WrappedExpression, Expression]],
+          p: Expression)(s: Statement): Statement = s match {
```

Comment:
I generally ascribe to the style the parameters make the line to long, you switch to 1 parameter per line (see: http://docs.scala-lang.org/style/declarations#classes). I've only ever seen this written in regards to class definitions (not functions) but I think following it for functions is a reasonable thing to do.
eg.

```scala
       def expandWhens(
            netlist: LinkedHashMap[WrappedExpression, Expression],
            defaults: Seq[mutable.Map[WrappedExpression, Expression]],
            p: Expression)
            (s: Statement): Statement = s match {
```

An alternative might be to draw inspiration from multiple parameter list style (http://docs.scala-lang.org/style/declarations#multiple-parameter-lists) and do something like
```scala
       def expandWhens(netlist: LinkedHashMap[WrappedExpression, Expression],
                       defaults: Seq[mutable.Map[WrappedExpression, Expression]],
                       p: Expression)
                      (s: Statement): Statement = s match {
```
Note that defaults and p are spaced to line up with the start of netlist while the parentheses of the parameter lists are also lined up